Cargo.toml

```diff
@@ -46,6 +46,7 @@ clap = { version = "3.1", features = ["derive"] }
 clippy_utils = { path = "clippy_utils" }
 derive-new = "0.5"
 if_chain = "1.0"
+indoc = "1.0"
```

Comment:
We can leave this out, don't need to have a test for it currently as the behaviour isn't fixed yet, when it is we'll use an `aux-build` proc macro or just the passthrough `macro_rules` to avoid an extra dependency